changelog/9114.feature.rst

```diff
@@ -0,0 +1 @@
+Added :ref:`pythonpath` setting that adds listed paths to `sys.path`.
```

Comment:
```suggestion
Added :ref:`pythonpath` setting that adds listed paths to `sys.path` for the duration of the test session.
```

Also WDYT about adding a note for existing pytest-pythonpath and pytest-srcpaths users? e.g.

> If you currently use the pytest-pythonpath or pytest-srcpaths plugins, you should be able to replace them with built-in `pythonpath` setting.

Reply:
I like it.